src/views/Entries/QualityControl/EntryCard/index.tsx

```diff
@@ -128,6 +128,16 @@ function EntryCard(props: EntryCardProps) {
         onEntryChange,
     } = props;
 
+    const [isVisible, setVisibility] = useState(false);
+
+    const handleShow = useCallback(() => {
+        setVisibility(true);
+    }, []);
+
+    const handleHide = useCallback(() => {
+        setVisibility(false);
+    }, []);
```

Comment:
We should have a hook for this called useStateModal

Reply:
I think @AdityaKhatri  and I had this discussion. useStateModal has not been imported to DEEP (or at least not in all branches then) which is why we used this hook instead.